### PR TITLE
Feature | Getters for Response Status Code & Data Length Properties

### PR DESCRIPTION
**Pull request by EstebanBorai**
Implements two methods on `Response` `struct`.

1. `status_code`
2. `data_length`

To act as "Getters" for these private properties in case they are needed by the client.

Im using these methods in an HTTP Server I'm writing, these could help me get both values for logging purposes.

### CODE REVIEW

File: src/response.rs [hunk 1]
```diff
@@ -436,6 +436,16 @@ where
 
         Ok(())
     }
+
+    /// Retrieves the current value of the `Response` status code
+    pub fn get_status_code(&self) -> StatusCode {
```
Comment (rawler) [inline src/response.rs:441]: I think the "get_" prefix is not used anywhere else in the public API:s of this codebase. `status_code` and `data_length` would do fine

File: src/response.rs [hunk 2]
```diff
@@ -436,6 +436,16 @@ where
...
+        self.status_code.clone()
+    }
+
+    /// Retrieves the current value of the `Response` data length
+    pub fn get_data_length(&self) -> Option<usize> {
+        self.data_length.clone()
```
Comment (rawler) [inline src/response.rs:447]: `.clone()` is superfluous here. `Option<usize>` is `Copy` https://doc.rust-lang.org/std/option/enum.Option.html#impl-Copy

**Review by rawler — CHANGES_REQUESTED** [review]
I'm not one of the regular maintainers here, but since your PR has been long without review, I'll have a go at it.

File: src/response.rs (hunk 1 above)
Comment (EstebanBorai) [reply src/response.rs:441]: Sure thing! I will replace those with the names suggested!

File: src/response.rs (hunk 2 above)
Comment (EstebanBorai) [reply src/response.rs:447]: Ohh! Thanks for sharing! Fixing it right away!

**Review by rawler — APPROVED** [review]

**Pull request merged**